File: 0217-contains-duplicate/0217-contains-duplicate.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function(nums) {
    let mapp=new Map();
    for(let i=0;i<nums.length;i++){
        if(mapp.has(nums[i]))return true;
     mapp.set(nums[i],1);
    }
    return false;
};

File: 0217-contains-duplicate/0217-contains-duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        map<int,int> mapp;
      for(int i=0;i<nums.size();i++){
          if(mapp.find(nums[i])!=mapp.end())return true;
          
       ++mapp[nums[i]];
      }
      return false;
    }
};